shell.nix

```diff
@@ -1,44 +0,0 @@
-{ pkgs ? import ./pkgs.nix {}, ci ? false }:
-
-with pkgs;
-let
-  utils = callPackage ./utils.nix {};
-in
-  mkShell {
-    nativeBuildInputs = [
-      nodejs
-      prefetch-npm-deps
-      shellcheck
```

Comment:
Where are these tools specified in the flake?

We still need most of these in the case of `nix develop`.

Reply:
It's defined in the flake. Only `prefetch-npm-deps` was missing and that has been fixed now.